### PR TITLE
docs(docs-infra): fix typos and spelling in the stackblitz-builder/README.md

### PR DESCRIPTION
**Pull request by arturovt**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/tools/stackblitz-builder/README.md [hunk 1]
```diff
@@ -53,7 +53,6 @@ example.
 
 ## Appendix: Why not generating stackblitz at runtime?
 
-At AngularJS, all the plunker examples were generated a runtime. The downside was that all the example code had to be 
-deployed as well and would no longer be useful after the plunker was generated. 
+At AngularJS, all the plunker examples were generated at runtime. The downside was that all the example code had to be deployed as well and would no longer be useful after the plunker was generated.
```
Comment (gkalpak) [inline aio/tools/stackblitz-builder/README.md]: Why this change (given that the rest of the file uses a different convention.

Comment (arturovt) [reply aio/tools/stackblitz-builder/README.md]: Hey, I will fix it. There is actually a typo fix, but probably I clicked "backspace" by accident :slightly_smiling_face:

Comment (arturovt) [reply aio/tools/stackblitz-builder/README.md]: @gkalpak reverted.

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
~~Can you amend the second commit to be a "fixup!" commit? That way it will be squashed into the original commit when it is merged.~~

I see you squashed them yourself already :-)

**Review by petebacondarwin — APPROVED** [review]

**Comment by arturovt** [comment]
> ~Can you amend the second commit to be a "fixup!" commit? That way it will be squashed into the original commit when it is merged.~
> 
> I see you squashed them yourself already :-)

:joy: Didn't know about that trick!

**Review by gkalpak — APPROVED** [review]
Thx for making the changes, @arturovt 👍
On last thing: Can you please update the commit message to be `docs(docs-infra): ...`?

**Comment by arturovt** [comment]
> Thx for making the changes, @arturovt 
> On last thing: Can you please update the commit message to be `docs(docs-infra): ...`?

Changed the PR title and amended a commit.
